include/swift/Basic/StringExtras.h

```diff
@@ -430,11 +432,16 @@ StringRef matchLeadingTypeName(StringRef name, OmissionTypeName typeName);
 /// Describes a set of names with an inheritance relationship.
 class InheritedNameSet {
   const InheritedNameSet *Parent;
-  llvm::StringSet<> Names;
+
+  // FIXME: This is a StringMap rather than a StringSet because StringSet
```

Comment:
Does LLVM have an open bug report/pull request for this?

Reply:
I guess that's the prompt I need to go add it.

Reply:
It was a genuine question, not a rhetorical question. 😄 Initially my thought was "oh, why even have a FIXME for this if we can't fix it", then I went through the docs and saw StringSet is defined as a wrapper around `StringMap<char>`, so adding it would be relatively easy.

Reply:
Yeah, no bug report, but not actually hard to do either.

Reply:
Turns out someone already did it and it got pulled in in the latest merge from upstream. :-)